data-explorer/kusto/api/netfx/request-properties.md

```diff
@@ -103,6 +103,22 @@ In the REST API, query parameters appear in the same JSON-encoded string as the
 
 The following example shows sample client code for using request properties:
 
+#### Json body
+```json
+{
+    "db": "databaseName",
+    "csl": "QueryHistory | where Duration > 4m and ClientActivityId contains 'unspecified'",
+    "properties": {
+        "Options": {
+            "maxmemoryconsumptionperiterator": 68719476736,
+            "max_memory_consumption_per_query_per_node": 68719476736,
+            "servertimeout": "50m"
+        }
```

Comment:
You might want to add "Parameters" here as well (same level as "Options", indicating query parameters)